### PR TITLE
Add dashboard widget doc

### PR DESCRIPTION
**Pull request by ngilain**

### CODE REVIEW

File: en/developer_guide/dashboard_widgets.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
+Dashboard widgets
+=================
+
+The Back office dashboard is the place to show to the user information concerning him. It's like a
```
Comment (alavieille) [inline en/developer_guide/dashboard_widgets.rst]: Back Office

File: en/developer_guide/dashboard_widgets.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+blackoard on which are pinned post-it. Each post-it is a widget. Open Orchestra comes with four
+widgets:
+
+* My 10 latest published nodes
+* My 10 latest nodes
+* My 10 latest published contents
+* My 10 latest contents
+
+As a developper, you can customize your application dashboard by choosing widgets and/or create new
```
Comment (alavieille) [inline en/developer_guide/dashboard_widgets.rst]: developer

File: en/developer_guide/dashboard_widgets.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+ones. Here is how to do.
+
+Configuring the dashboard
+-------------------------
+To choose the widgets to display in the dashboard, add a key ``open_orchestra_backoffice.dashboard_widgets``
+to your app configuration. Then set the list of widgets you want to use. Here is an exemple in yaml
+format showing the 4 standard widgets:
+
+.. code-block:: yml
```
Comment (alavieille) [inline en/developer_guide/dashboard_widgets.rst]: use code-block yaml : .. code-block:: yaml

File: en/developer_guide/dashboard_widgets.rst [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    open_orchestra_backoffice:
+        dashboard_widgets:
+        - last_nodes
+        - draft_nodes
+        - last_contents
+        - draft_contents
+
+Creating a widget
+-----------------
+This operation is very simple. All you have to do is create a Backbone view and register it in
```
Comment (itkg-alavieille) [inline en/developer_guide/dashboard_widgets.rst]: may be a link to this documentation which explain how register a view
https://github.com/open-orchestra/open-orchestra-docs/blob/master/en/developer_guide/specific_backbone_view.rst

File: en/developer_guide/dashboard_widgets.rst [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+-------------------------
+To choose the widgets to display in the dashboard, add a key ``open_orchestra_backoffice.dashboard_widgets``
+to your app configuration. Then set the list of widgets you want to use. Here is an exemple in yaml
+format showing the 4 standard widgets:
+
+.. code-block:: yml
+
+    open_orchestra_backoffice:
+        dashboard_widgets:
+        - last_nodes
```
Comment (nicolasThal) [inline en/developer_guide/dashboard_widgets.rst]: you should correct the indentation

File: en/developer_guide/dashboard_widgets.rst (hunk 4 above)
Comment (nicolasThal) [inline en/developer_guide/dashboard_widgets.rst]: I would suppress the `This operation is very simple`

File: en/developer_guide/dashboard_widgets.rst [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+        - draft_contents
+
+Creating a widget
+-----------------
+This operation is very simple. All you have to do is create a Backbone view and register it in
+Open Orchestra.
+
+Creating a backbone view
+~~~~~~~~~~~~~~~~~~~~~~~~
+The Back office is driven by Backbone. All widgets are Backbone views added automatically to the
```
Comment (nicolasThal) [inline en/developer_guide/dashboard_widgets.rst]: As the Back Office is build using `BackboneJs`, all widgets are Backbone views which are added to the dashboard view.

**Comment by ngilain** [comment]
@itkg-alavieille , @nicolasThal : doc updated

File: en/developer_guide/dashboard_widgets.rst [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+            - last_contents
+            - draft_contents
+
+Creating a widget
+-----------------
+To build a new widget, you only have to create its Backbone view and register it in Open Orchestra.
+
+Creating a backbone view
+~~~~~~~~~~~~~~~~~~~~~~~~
+As the Back Office is build with BackboneJs, all widgets are Backbone views added to the main
```
Comment (nicolasThal) [inline en/developer_guide/dashboard_widgets.rst]: Can you add `` around BackboneJs ?

File: en/developer_guide/dashboard_widgets.rst [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+your view from scratch or extend one of the pseudo-abstract layers used by the standard Open
```
Comment (nicolasThal) [inline en/developer_guide/dashboard_widgets.rst]: added to the main your view ?

I think there are word to suppress :)

File: en/developer_guide/dashboard_widgets.rst [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+Orchestra widgets. To get some examples, you can have a look at the files located in
```
Comment (nicolasThal) [inline en/developer_guide/dashboard_widgets.rst]: you can take a look

File: en/developer_guide/dashboard_widgets.rst [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+``BackOfficeBundle/Ressources/public/coffee/dashboard/widget``
+
+Your widget html can be anything, but we suggest you to use the jarvis widget template to have a
+visual uniformity in the dashboard. This template can be found in
+``BackOfficeBundle/Ressources/views/BackOffice/Underscore/dashboard/widget/abstract/abstractWidgetListView._tpl.twig``
+
+Declaring the view
+~~~~~~~~~~~~~~~~~~
+To make your newly created widget available in Open Orchestra, you just need to add it to the
```
Comment (nicolasThal) [inline en/developer_guide/dashboard_widgets.rst]: you just need to => you have to 

have to give the obligation if I have a good memory :)

File: en/developer_guide/dashboard_widgets.rst [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+appConfigurationView. This object is used in the Back Office to list all backbone views and access
```
Comment (nicolasThal) [inline en/developer_guide/dashboard_widgets.rst]: appConfigurationView with ``

Comment (nicolasThal) [inline en/developer_guide/dashboard_widgets.rst]: Is there a documentation where we talk about the appConfigurationView ?

If yes, a link to this page could be good (the little summary should stay)

Comment (ngilain) [inline en/developer_guide/dashboard_widgets.rst]: @nicolasThal : I think there's no doc about this, I suggest to create a trello task about this

**Comment by ngilain** [comment]
@nicolasThal : update done

**Comment by nicolasThal** [comment]
:+1:

**Comment by alavieille** [comment]
:+1:

**Pull request merged**